myproject/templates/edit_entry.jinja2

```diff
@@ -0,0 +1,16 @@
+{% extends "layout.jinja2" %}
+{% block body %}
+
+
+<form>
+    Title:<br>
```

Comment:
I would urge you to put labels for form fields into `<label>` elements.  It allows you to style them and interact with them in ways that will prove useful.  Plus it helps screen readers because you can say that a given label belongs with a given input.  That's enormously helpful for accessibility.